tools/mitohifi/macros.xml

```diff
@@ -11,12 +11,12 @@
     </xml>
     <xml name="requirements">
         <requirements>
-            <container type="docker">quay.io/bgruening/mitohifi:2_galaxy0</container>
+            <container type="docker">ghcr.io/marcelauliano/mitohifi:master</container>
```

Comment:
@abueg let me know when you are happy with this Container. I will archive it and provide a version for reproducibility.

Happy holidays.

Reply:
OK, i'll check with Marcela to see if the current version of the master is good for archiving, and get back to you 👍🏼 thank you, and happy holidays! 🐰

Reply:
hi @bgruening ! Marcela said the current version of master is good to go for archiving 👍🏼

Reply:
Ok, I'm pushing it to quay.io atm.

The tests are failing here at the moment.

Reply:
quay.io/bgruening/mitohifi:3_galaxy0 is now available :)